### ISSUE TITLE
No whitespace after if/for/foreach/while etc

### ISSUE THREAD

**Issue by DominikTo** [fruux/sabre-cs#6]
We don't like these whitespaces:
- `if ()`
- `for ()`
- `foreach ()`
- `while ()`
- `catch ()`
- `function ()`
- ...

**Comment by evert** [comment]
This is not true. I'm pretty sure you accidentally ran the PSR-2 fixer script. Our CS fixer does not change this (but it doesn't enforce anything either).

**Comment by DominikTo** [comment]
Should we enforce it then? :)

**Comment by Hywan** [comment]
But… it's not consistent everywhere. So we should create a new fixer for that :stuck_out_tongue:.

**Comment by DominikTo** [comment]
Yup, would be nice to have it cleaned up everywhere :)

**Comment by evert** [comment]
:+1: for a fixer ;)

**Comment by evert** [comment]
Ok @fruux/developers, this aint right.

For some reason I:
- Never have a space after `while`, `foreach`, `do`, `for`.
- always have a space after `if`.

I don't know why either, but removing the space after the `if` looks very ugly to me :S It would be a bit weird to exclude `if` from the list, so how about we do the opposite anyway? Consistently add a space after _every_ control structure.

**Comment by DominikTo** [comment]
As long as it's consistent, it's fine for me. :)

There may be a built-in fixer already, that is able to do that.

**Comment by ahackschmitz** [comment]
+1 for adding space!

**Comment by Hywan** [comment]
Spaces everywhere please. Else, it's not consistent :-].

**Comment by DominikTo** [comment]
I think you guys need to make up your mind about the rules first, make a final decision and then someone should implement it. ;)

/cc @fruux/developers

**Comment by Hywan** [comment]
Simple. Make a vote.
1. whitespace after structure controls, so:
   
   ``` php
   if ()
   elseif ()
   foreach ()
   switch ()
   while ()
   …
   ```
2. no whitespace after structure controls, so:
   
   ``` php
   if()
   elseif()
   foreach()
   switch()
   while()
   …
   ```

**Comment by Hywan** [comment]
I vote :one:.

**Comment by ahackschmitz** [comment]
I vote :one:.

And if I understand @evert comment correctly, he would go for it, too. (If only he doesn't want to remove the whitespace in `if ()`:

> I don't know why either, but removing the space after the if looks very ugly to me :S It would be a bit weird to exclude if from the list, so how about we do the opposite anyway? Consistently add a space after every control structure.

**Comment by DominikTo** [comment]
If it ends up being :one:, there's #15.

**Comment by evert** [comment]
Also vote 1

**Comment by DominikTo** [comment]
Closed via #15.
